portfolio/src/main/webapp/misc.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lina's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <ul>
+      <li><a href="index.html">Home</a></li>
+      <li><a href="misc.html">For Fun</a></li>
+    </ul>
+    <div id="content">
+      <h1 id="title">"I am large, I contain multitudes" - Walt Whitman, <i>Song of Myself</i></h1>
+      <p>Music has always held a very special place in my heart. It's a place I can escape to during the buzz of everyday life and responsibilites.</p> 
+      <p>My music taste is extremely varied. I listen to a lot of older Colombian music that I grew up with.</p>
+      <p>I also listen to a lot of reggaeton, and a personal favorite artist of mine is Bad Bunny.</p>
+      <p>My current top artists are Frank Ocean, Tyler the Creator, Omar Apollo, and Megan Thee Stallion.</p>
+      <p>Here's a preview of what I'm currently listening to!</p>
+      <p>These are a couple of my top albums of all time:</p>
+    </div>
+    <div class="center">
+        <iframe src="https://open.spotify.com/embed/album/5lJqux7orBlA1QzyiBGti1"
+        width="300" height="380" frameborder="0" allowtransparency="true"
+        allow="encrypted-media"></iframe>
+        <iframe src="https://open.spotify.com/embed/album/3yuV4GeplvzBIiYGhQp2nd"
+        width="300" height="380" frameborder="0" allowtransparency="true"
+        allow="encrypted-media"></iframe>
+        <iframe src="https://open.spotify.com/embed/album/392p3shh2jkxUxY2VHvlH8" 
+        width="300" height="380" frameborder="0" allowtransparency="true"
+        allow="encrypted-media"></iframe>
+        <iframe src="https://open.spotify.com/embed/album/3mH6qwIy9crq0I9YQbOuDf" 
+        width="300" height="380" frameborder="0" allowtransparency="true" 
+        allow="encrypted-media"></iframe>
+        <!-- <iframe src="https://open.spotify.com/embed/album/4UlGauD7ROb3YbVOFMgW5u" 
```

Comment:
In general, it's not good to submit commented out code in a PR unless there's a good reason. Is there a reason you want to keep this around but hidden?